sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario22/PDSPrepareIntegrationScenario22IntTest.java

```diff
@@ -225,4 +228,33 @@ private void approveJobAndSimulateSourceCodeAvailable(TestProject project, UUID
           /* @formatter:on */
     }
 
+    @Test
+    public void startPDSPrepareJobFromRemoteCodeScanConfigurationwithoutClient() {
+        /* @formatter:off */
+
+        /* prepare */
```

Comment:
Integration tests do need much time for execution.
Here is currently only tested that the remote setup does always lead to preparation execution.

We should test here that the script on PDS side is able to fetch the configuration (e.g. return an info message with the location and the type and check the message inside the test). The test would make more sense in my opinion (because we test that the remote configuration is available at script execution time)

Reply:
We could also create a custom ENV variable, put the complete config into the custom variable, dump it via the test methods and fetch the information inside the integration test via TestAPI methods.

Then we could create a config model from this (json content) and test for 
- all content as expected
- location, type etc. are available (remote connection data)

So it would be clear that all necessary data for preparation is available at PDS side